core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -188,24 +211,43 @@ async fn get_wallet(address: &str) -> Result<Wallet<YagnaEthSigner>, GenericErro
     Ok(wallet)
 }
 
-async fn unlock_wallet<S: EthereumSigner + Clone>(wallet: Wallet<S>) -> Result<(), GenericError> {
+fn get_zk_network(network: Network) -> ZkNetwork {
```

Comment:
Not sure if this deserves to be a separate function.

Reply:
its only nice for the "central" default value, now it makes little sense ( only called 2x )